python/tests/test_dataset.py

```diff
@@ -335,31 +357,35 @@ def test_coords_proxy_comparison_operators():
 def test_sum_mean():
     d = sc.Dataset(
         {
-            'a': sc.Variable([Dim.X, Dim.Y],
-                             values=np.arange(6, dtype=np.int64)
-                             .reshape(2, 3)),
-            'b': sc.Variable([Dim.Y], values=np.arange(3, dtype=np.int64))
+            'a':
+            sc.Variable([Dim.X, Dim.Y],
+                        values=np.arange(6, dtype=np.int64).reshape(2, 3)),
+            'b':
+            sc.Variable([Dim.Y], values=np.arange(3, dtype=np.int64))
         },
         coords={
             Dim.X: sc.Variable([Dim.X], values=np.arange(2, dtype=np.int64)),
             Dim.Y: sc.Variable([Dim.Y], values=np.arange(3, dtype=np.int64))
         },
         labels={
-            "l1": sc.Variable([Dim.X, Dim.Y],
-                              values=np.arange(6, dtype=np.int64)
-                              .reshape(2, 3)),
-            "l2": sc.Variable([Dim.X], values=np.arange(2, dtype=np.int64))
+            "l1":
+            sc.Variable([Dim.X, Dim.Y],
+                        values=np.arange(6, dtype=np.int64).reshape(2, 3)),
+            "l2":
+            sc.Variable([Dim.X], values=np.arange(2, dtype=np.int64))
         })
     d_ref = sc.Dataset(
         {
-            'a': sc.Variable([Dim.X],
-                             values=np.array([3, 12], dtype=np.int64)),
+            'a': sc.Variable([Dim.X], values=np.array([3, 12],
+                                                      dtype=np.int64)),
             'b': sc.Variable(3)
         },
-        coords={Dim.X: sc.Variable([Dim.X],
-                                   values=np.arange(2, dtype=np.int64))},
-        labels={"l2": sc.Variable([Dim.X],
-                                  values=np.arange(2, dtype=np.int64))})
+        coords={
+            Dim.X: sc.Variable([Dim.X], values=np.arange(2, dtype=np.int64))
+        },
+        labels={
+            "l2": sc.Variable([Dim.X], values=np.arange(2, dtype=np.int64))
+        })
```

Comment:
Maybe we should agree on what formatting tool to use to avoid this noise in the diff? I have been using `yapf`.

Reply:
Me too. I've been using `pep8 style`, maybe there are version differences. In any case i'll undo un related formatting changes.